cartography/intel/aws/ec2/route_tables.py

```diff
@@ -0,0 +1,123 @@
+import time
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+from cloudconsolelink.clouds.aws import AWSLinker
+
+from botocore.exceptions import ClientError
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+aws_console_link = AWSLinker()
+
+
+@timeit
+@aws_handle_regions
+def get_route_tables_data(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    client = boto3_session.client('ec2', region_name=region)
+    try:
+        paginator = client.get_paginator('describe_route_tables')
+        route_tables: List[Dict] = []
+        for page in paginator.paginate():
+            route_tables.extend(page['RouteTables'])
+        for route_table in route_tables:
+            route_table['region'] = region
+            route_table['consolelink'] = ''  # TODO
+    except ClientError as e:
+        if e.response['Error']['Code'] == 'AccessDeniedException' or e.response['Error']['Code'] == 'UnauthorizedOperation':
+            logger.warning(
+                f'ec2:describe_subnets failed with AccessDeniedException; continuing sync.',
+                exc_info=True,
+            )
+        else:
+            raise
+    return route_tables
+
+
+@timeit
+def load_route_tables(
+    neo4j_session: neo4j.Session, data: List[Dict], aws_account_id: str,
+    aws_update_tag: int,
+) -> None:
+    ingest_route_tables = """
+    UNWIND $route_tables as route_table
+    MERGE (rtab: EC2RouteTable{id: route_table.RouteTableId})
```

Comment:
generate ARN for EC2RouteTable in the format `arn:${Partition}:ec2:${Region}:${Account}:route-table/${RouteTableId}`